manager/state/raft/transport/transport_test.go

```diff
@@ -7,19 +7,57 @@ import (
 	"golang.org/x/net/context"
 
 	"github.com/coreos/etcd/raft"
+
 	"github.com/coreos/etcd/raft/raftpb"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+// Build a snapshot message where each byte in the data is of the value (index % sizeof(byte))
+func getSnapshotMessage(from uint64, to uint64) raftpb.Message {
```

Comment:
Trying to use this from testutils causes a circular dependency and moving this test to a transport_test package was just too problematic since unexported fields have been used extensively in the tests. Leaving as is for now, will find a way to address this in the future.

Reply:
why do you need to use this from `testutils`?

Reply:
Keeping it in once place instead of duplicating it. It's also used in raft_test.go